packages/example-log-extension/test/acceptance/log.extension.acceptance.ts

```diff
@@ -317,11 +318,11 @@ describe('log extension acceptance test', () => {
     return [2, 2];
   }
 
-  function createConsoleSpy() {
-    spy = sinon.spy(console, 'log');
+  function createLogSpy() {
+    spy = sinon.spy(LogActionProvider.prototype, 'log');
   }
 
-  function restoreConsoleSpy() {
+  function restoreLogSpy() {
```

Comment:
This code is duplicated here and in `p`ackages/example-log-extension/test/unit/providers/log-action.provider.unit.ts`, please move both `create` and `restore` helpers to a shared file.